tests/unit_test/surveymonkey/surveymonkey_etl_test.py

```diff
@@ -128,7 +128,7 @@ def test_should_extract_question_answers(self):
                         "rows": [{
                             "id": "ROW_ID",
                             "text": "This is the row",
-                            "weight": 0
+                            "weight": "3"
```

Comment:
This is my recommendation:
I mentioned it yesterday, I don't think we should convert numbers to a string. It basically changes is from a continuous value that one can average etc, to a categorical value.
If there are inconsistencies, it would be good to document them (e.g. in tests).
Alternatively just leave `weight` out until needed and we had more time at looking at various examples.

Reply:
OK

Reply:
`None` is causing problem (while creating schema and loading the data in BQ) if I don't add a default value in the `get` function? What can I add here as default value, for string values I was adding `""`. Or do you have any idea how can I manage the None value in json not to cause a problem.

```python
"id": answer_option.get("id",""),
"text": answer_option.get("text",""),
"type": answer_option.get("type",""),
"weight": answer_option.get("weight")
```